src/QuadForm/Herm/Lattices.jl

```diff
@@ -750,3 +750,114 @@ function maximal_integral_lattice(V::HermSpace)
   return maximal_integral_lattice(L)
 end
 
+################################################################################
+#
+#  Intersection
+#
+################################################################################
+
+#=
+    _reconstruction_herm_lattice(L::ZLat, E::Hecke.NfRel{nf_elem}, 
+    f::Hecke.VecSpaceRes{Hecke.NfRel{nf_elem}, Hecke.NfRelElem{nf_elem}}, Vas)
+                  -> HermLat{} 
+    
+Takes the $\mathbb{Z}$-lattice $L$ and reconstructs a herm lattice with base field
+$E$ and ambient space $Vas$. 
+=#
+function _reconstruction_herm_lattice(L::ZLat, E::Hecke.NfRel{nf_elem}, f::Hecke.VecSpaceRes{Hecke.NfRel{nf_elem}, Hecke.NfRelElem{nf_elem}}, Vas)
+  M = basis_matrix(L)
+  OE = maximal_order(E)
+  n = absolute_degree(E)
+
+  K = base_field(E)
+  OK = maximal_order(K)
+  deg = div(ncols(M),n)
+  
+  a = zeros(E, deg, deg)
+  coeffs = (Hecke.fractional_ideal_type(OE))[]
+  
+  for i = 1:deg
+ 
+    ind = findfirst(k -> !iszero(M[k,:]), 1+(i-1)*n:i*n) + (i-1)*n
+    if typeof(ind) == Nothing
```

Comment:
```suggestion
    if typeof(ind) === nothing
```
I guess `Nothing` is a type, `nothing` is an object